### PR TITLE
Add a way to reset rolepayday amounts

### LINKED ISSUE #4661: [Bank] Allow to reset ` rolepaydayamount` for a certain role
**Describe the enhancement**
Allow a command to reset a specific role (or multiple roles at once) from the command `[p]economyset rolepaydayamount role amount`. There is not currently a way to effectively do this.

**Does this enhancement break existing functionality?**
 Yes
 **No**

### PR DESCRIPTION
**Pull request by Danstr5544**
## Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
Fixes #4661 by allowing you to reset a roles payday amount by setting the value to/below 0.

### CODE REVIEW

File: redbot/cogs/economy/economy.py
Comment (Jackenmen) [inline redbot/cogs/economy/economy.py]: We try to stick to max line length in strings too, but Black doesn't automatically handle those yet.
```suggestion
                    "The bank requires that you set the payday to be less than"
                    " its maximum balance of {maxbal}."
```

File: redbot/cogs/economy/economy.py
Comment (Jackenmen) [inline redbot/cogs/economy/economy.py]: typo
```suggestion
                        "Users with this role will now receive the default pay "
```

File: redbot/cogs/economy/economy.py
Comment (Jackenmen) [inline redbot/cogs/economy/economy.py:979]: This needs to also be updated in `docs/cog_guides/economy.rst` file.

**Review by Jackenmen — CHANGES_REQUESTED** [review]
The PR seems to be mostly fine, just needs a few minor changes before approval.

File: redbot/cogs/economy/economy.py
Comment (Danstr5544) [reply redbot/cogs/economy/economy.py:979]: Resolved in latest PR

**Review by Jackenmen — APPROVED** [review]
Looks good to me now, thanks for PRing.

**Pull request merged**
